client/Configurator.c

```diff
@@ -216,12 +216,19 @@ static void udpInterface(Dict* config, struct Context* ctx)
                 key = String_clone(key, perCallAlloc);
                 char* lastColon = CString_strrchr(key->bytes, ':');
 
-                if (!Sockaddr_parse(key->bytes, NULL)) {
-                    // it's a sockaddr, fall through
-                } else if (lastColon) {
-                    // try it as a hostname.
-                    Log_critical(ctx->logger, "Couldn't add connection [%s], "
-                                                "hostnames aren't supported.", key->bytes);
+                if (lastColon) {
+                    if (!Sockaddr_parse(key->bytes, NULL)) {
```

Comment:
I'm not sure this is correct, this assumes `Sockaddr` has a colon in it, I don't know if that's the case. Assuming it's the [sockaddr struct][1] there's a good chance it doesn't contain a colon but is still valid. I would appreciate if somebody more familiar with that part of the source could confirm this :)

[1]: http://www.retran.com/beej/sockaddr_inman.html